### ISSUE TITLE
Add projects to project list

### ISSUE THREAD

**Issue by jelletreep** [UtrechtUniversity/research-engineering#1]
Create a list of projects. How should it look like? Plain list with links to project repositories? Short descriptions? Visuals?

**Comment by jelletreep** [comment]
Project includes the following information:
Name (and link to github repo)
Short description
Faculty
Technologies
Project duration

**Comment by qubixes** [comment]
| [ASReview](https://github.com/asreview/asreview) | 
| --- | 
| 2018-2021 |
| Faculty of Social Sciences |
| Machine learning, active learning, Python, flask, hyperparameter optimization, simulation | 
| Raoul Schram, Parisa Zahedi |
| ASReview is a machine learning tool to aid researchers in performing systematic reviews. It uses active learning to present users with more likely relevant papers. It has been written in Python 3.7+, and hyper parameters have been optimized using the hyperopt package. We have also contributed to the initial back-end for the user interface using flask. | ```

| [Streetview 1](https://github.com/UtrechtUniversity/streetview-greenery) |
| --- | 
| 2019-2021 |
| Faculty of Geosciences |
| Machine learning, Python, image segmentation, deeplab, kriging, geolocation, CityScapes | 
| Raoul Schram |
| For the streetview project we have used the (formerly) open street view data from the municipality of Amsterdam to create a map of the greenness. This is done by taking the images and segmenting each image into different classes. The number of pixels in each image belonging to the "greenery" class is used to create the Amsterdam greenery map. | ```


| [Network Entropy](https://github.com/qubixes/temporal-network-synthesis) |
| --- | 
| 2020-2021 |
| Department of Information and Computing Sciences |
| Temporal networks, Python, numba, simulation | 
| Raoul Schram |
| To improve the theoretical analysis and comparison of different temporal networks, we have invented a new metric to study them. The measure is called network entropy, and is applicable to any temporal network. We showed with simulations that processes on a network behave very differently, depending on the network entropy. | ```

| [Protosc](https://github.com/UtrechtUniversity/protosc) |
| --- | 
| 2020-2021 |
| Faculty of Social Sciences |
| Feature selection, Python, image classification, wrapper, filter, genetic algorithm | 
| Raoul Schram |
| Protosc is a Python library that aims to determine which features are relevant to a given classification problem. It does so by using wrapper/filter/genetic algorithms, after which automatic statistical analysis is used to determine which features are significant. The package also includes a few different options for an image classification pipeline. | ```

**Comment by qubixes** [comment]
Btw, should we add former members to the RSE field? Could be confusing for people looking for contact to a specific RSE?

**Comment by MartineDeVos** [comment]
> 

If a former member worked on a project, I think he/she should also be in the RSE field. Additionally, we could add a former members sublist to the "meet the team" section

**Comment by RoelBrouwer** [comment]
Another point of discussion for the recap (maybe): should the 'technologies' field be limited to technologies/expertise that we as research engineers developed/used, or should it rather be a list of keywords describing the project?

Also, do we refer to ourselves as RSEs, REs, Research Engineers, or anything else? Would be good to be consistent and clear about that :)

**Comment by RoelBrouwer** [comment]
```
| [Ocean Parcels Numba](https://github.com/OceanParcels/parcels) | 
| --- | 
| 01/07/2021 - now |
| Faculty of Science |
| Technologies: Python, numba | 
| RSEs: [Roel Brouwer](uu.nl/staff/RJJBrouwer) and Raoul Schram |
| Investigating the feasibility of speeding up existing Python code for [Parcels](oceanparcels.org) using numba. The aim is to speed up the simulation enough to eliminate the need for a separate (partial) JIT/C path in the code. This should lead to a more flexible and maintainable code base. |

| [Ocean Parcels particle-particle interaction](https://github.com/OceanParcels/parcels) | 
| --- | 
| 01/07/2020 - 30/06/2021 |
| Faculty of Science |
| Technologies: Python, datastructures, simulation | 
| RSEs: [Roel Brouwer](uu.nl/staff/RJJBrouwer) and Raoul Schram |
| Providing a working implementation of particle-particle interaction for [Parcels](oceanparcels.org). The aim was to allow simulated particles to interact and influence each others states. This project involved reviewing and partially restructuring the data structures that Parcels uses for storing particle data, and implementing particle-particle interaction under certain conditions. |
```

**Comment by MartineDeVos** [comment]
> > 
> 
> If a former member worked on a project, I think he/she should also be in the RSE field. Additionally, we could add a former members sublist to the "meet the team" section

If a former member does not have an UU profile, we may add a linkt to the GitHub profile instead

**Comment by MartineDeVos** [comment]
> 



> Another point of discussion for the recap (maybe): should the 'technologies' field be limited to technologies/expertise that we as research engineers developed/used, or should it rather be a list of keywords describing the project?
 
I would be in favour of keywords. Technologies are surely interesting to mention but may be to limited to describe the topic (for example sensitive data or workflow/pipeline) of a project.

**Comment by jelletreep** [comment]
I would prefer technologies/expertise that we as research engineers developed/used, as the next field is a short project description, so if we describe the project in keywords that is double. In the former case you can provide 'new' info in the technology field.

**Comment by jelletreep** [comment]
```
| [Animal Sounds](https://github.com/UtrechtUniversity/animal-sounds) | 
| --- | 
| 2019-2021 |
| Faculty of Science |
| Technologies: bioacoustics, audio, librosa, machine learning, deep learning, feature engineering | 
| RSEs: [Jelle Treep](https://www.uu.nl/staff/HJTreep), [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), Casper Kaandorp |
| We developed algorithms and a data processing workflow to detect vocalizations of Chimpanzees in a large body of audio data from the African tropical rainforest. The workflow consists of: 1) a filtering step where irrelevant audio data is removed to speed up manual annotation, 2) a feature engineering and feature selection step, and 3) classification using support vector machines and convolutional neural networks |
```

**Comment by jelletreep** [comment]
```
| [Animal Sounds](https://github.com/UtrechtUniversity/animal-sounds) | 
| --- | 
| 2019-2021 |
| Faculty of Science |
| Technologies: bioacoustics, audio, librosa, machine learning, deep learning, feature engineering | 
| RSEs: [Jelle Treep](https://www.uu.nl/staff/HJTreep), [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), Casper Kaandorp |
| We developed algorithms and a data processing workflow to detect vocalizations of Chimpanzees in a large body of audio data from the African tropical rainforest. The workflow consists of: 1) a filtering step where irrelevant audio data is removed to speed up manual annotation, 2) a feature engineering and feature selection step, and 3) classification using support vector machines and convolutional neural networks |
```

**Comment by hailihu** [comment]
| [Data Donation - proof of concept](https://github.com/eyra/port-poc) | 
| --- | 
| 04/2021-10/2021 |
| Faculty of Social Sciences, Human Data Science group |
| Technologies: Python, WebAssembly, Pyodide, Data Faker | 
| Research Engineers: Haili Hu, Roos Voorvaart |
| In this project, we collaborated with the [Human Data Science group](https://hds.sites.uu.nl/) and [Eyra](https://www.eyra.co/) to make data from social media platforms easily accessible to researchers, while preserving privacy. Respondents can voluntary donate their data download packages through an online platform (PORT), and researchers can provide custom data extraction scripts, which will be run locally on the respondent’s devices. A proof-of-concept PORT has been developed by Eyra, while data extraction scripts and fake data packages were provided by the Research Engineers team.| ```

**Comment by hailihu** [comment]
```| [Precision Nudging](https://github.com/UtrechtUniversity/nudging) | 
| --- | 
| 04/2021- |
| Faculty of Law, Economics and Governance, Public Governance and Management |
| Technologies: Python, scikit-learn, data simulation | 
| Research Engineers: Haili Hu, Raoul Schram|
| Changing behavior is necessary to tackle societal problems, such as obesity and financial problems. One way to change behavior is by nudging people. Anudge is a way to change behavior without prohibiting options or changing its costs. However, nudges are often one-size-fits-all techniques: everyone is offered the same nudge. The scientific aim of this project is to use open data to develop predictive models with Machine Learning, in order to determine the most effective nudge for persons, given the nudging goal and the individual personal circumstances.| ```

**Comment by MartineDeVos** [comment]
> | [Data Donation - proof of concept](https://github.com/eyra/port-poc) | 
> | --- | 
> | 04/2021-10/2021 |
> | Faculty of Social Sciences, Human Data Science group |
> | Technologies: Python, WebAssembly, Pyodide, Data Faker | 
> | Research Engineers: Haili Hu, Roos Voorvaart |
> | In this project, we collaborated with the [Human Data Science group](https://hds.sites.uu.nl/) and [Eyra](https://www.eyra.co/) to make data from social media platforms easily accessible to researchers, while preserving privacy. Respondents can voluntary donate their data download packages through an online platform (PORT), and researchers can provide custom data extraction scripts, which will be run locally on the respondent’s devices. A proof-of-concept PORT has been developed by Eyra, while data extraction scripts and fake data packages were provided by the Research Engineering team.|
> ```
@hailihu  you could add your publication as an additional item

**Comment by MartineDeVos** [comment]
| [Global Goals](https://github.com/UtrechtUniversity/global-goals) | 
| --- | 
| 10/2019-08/2020 |
| Faculty of Geosciences, Global Sustainability Governance|
| Technologies: Webscraping,Python,AWS,terraform| 
| RSEs:,[Martine de Vos](https://www.uu.nl/staff/MgdeVos)|
| The global goals project investigates the effect of the United Nations' Sustainable Development Goals (SDGs) on the global network of intergovernmental organizations. This network is represented by the hyperlinks on the organizations’ websites. We have retrieved the historical - from 2012 up to 2019- hyperlinks for a given set of international organizations via the Internet Archive.| ```

**Comment by MartineDeVos** [comment]
| [Animal Sounds](https://github.com/UtrechtUniversity/animal-sounds) | 
| --- | 
| 2019-2021 |
| Faculty of Science |
| Technologies: bioacoustics, audio, librosa, machine learning, deep learning, feature engineering | 
| RSEs: [Jelle Treep](https://www.uu.nl/staff/HJTreep), [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), Casper Kaandorp |
| We developed algorithms and a data processing workflow to detect vocalizations of Chimpanzees in a large body of audio data from the African tropical rainforest. The workflow consists of: 1) a filtering step where irrelevant audio data is removed to speed up manual annotation, 2) a feature engineering and feature selection step, and 3) classification using support vector machines and convolutional neural networks |```

**Comment by MartineDeVos** [comment]
``| [Global Goals](https://github.com/UtrechtUniversity/global-goals) | 
| --- | 
| 10/2019-08/2020 |
| Faculty of Geosciences, Global Sustainability Governance|
| Technologies: Webscraping,Python,AWS,terraform| 
| RSEs:,[Martine de Vos](https://www.uu.nl/staff/MgdeVos)|
| The global goals project investigates the effect of the United Nations' Sustainable Development Goals (SDGs) on the global network of intergovernmental organizations. This network is represented by the hyperlinks on the organizations’ websites. We have retrieved the historical - from 2012 up to 2019- hyperlinks for a given set of international organizations via the Internet Archive.| ```

**Comment by MartineDeVos** [comment]
```| [Global Goals](https://github.com/UtrechtUniversity/global-goals) | 
| --- | 
| 10/2019-08/2020 |
| Faculty of Geosciences, Global Sustainability Governance|
| Technologies: Webscraping,Python,AWS,terraform| 
| RSEs:[Jelle Treep](https://www.uu.nl/staff/HJTreep),[Martine de Vos](https://www.uu.nl/staff/MgdeVos)|
| The global goals project investigates the effect of the United Nations' Sustainable Development Goals (SDGs) on the global network of intergovernmental organizations. This network is represented by the hyperlinks on the organizations’ websites. We have retrieved the historical - from 2012 up to 2019- hyperlinks for a given set of international organizations via the Internet Archive.| ```

**Comment by MartineDeVos** [comment]
```| [Data donation – Anonymize Instagram](https://github.com/UtrechtUniversity/anonymize-ddp) | 
| --- | 
| 01/2020-10/2021 |
| Faculty of Social Sciences, Human Data Science group|
| Technologies: De-identification, Python, detecting and blurring visual content: Pytorch, Imaging Library,EAST model,OpenCV| 
| RSEs: [Roos Voorvaart](https://github.com/RVoor), [Martine de Vos](https://www.uu.nl/staff/MGdeVos)|
| Publications: [Automatic de-identification of data download packages] 
(Automatic de-identification of data download packages)
[ Open data set for validation](https://zenodo.org/record/4472606#.YaSvDLso9hE)|
| Instagram users can request their personal data in a Data Download Package (DDP). The data in DDPs may be used for social science research,but can be deeply private. To protect the privacy of research participants, we have developed a de-identification algorithm that is able to handle typical characteristics of DDPs.|```

**Comment by MartineDeVos** [comment]
```
| [Crunchbase](https://github.com/UtrechtUniversity/ia-webscraping) | 
| --- | 
| 05/2020- |
| Faculty of Geosciences, Dynamics of Innovation Systems|
| Technologies: Webscraping,Internet Archive,Pipeline,Internet,Python,AWS,Terraform| 
| RSEs:[Casper Kaandorp](),[Martine de Vos](https://www.uu.nl/staff/MgdeVos)|
| The Crunchbase project assesses the sustainability of European startup-companies by analyzing their websites. As the researcher is interested in the pre-Corona situation, we scrape webpages from the Internet Archive. Together with [SURF](https://www.surf.nl/en/custom-cloud-solutions) we have developed a method to set up AWS workflow for collecting and analyzing these webpages.| ```

**Comment by MartineDeVos** [comment]
```| [SummerFAIR](https://github.com/UtrechtUniversity/anonymize-ddp) | 
| --- | 
| 02/2021- |
| Faculty of Veterinary Medicine, Veterinary Epidemiology |
| Technologies: Semantic Web,Ontologies,RDF,Docker,Python,R| 
| RSEs:[Martine de Vos](https://www.uu.nl/staff/MGdeVos)|
| The summerFAIR project aims to integrate existing data sets on transmission experiments to enable reanalysis and meta-analysis. We have developed a pipeline based to map data to a shared vocabulary, convert them to linked data triples and perform integrated analyses.| ```

**Comment by MartineDeVos** [comment]
```
| [Data privacy](https://github.com/UtrechtUniversity/dataprivacyhandbook) | 
| --- | 
| 10/2021- |
| Research Data Management Support|
| Technologies: GDPR, de-identification, synthetic data, federated analysis| 
| RSEs:[Martine de Vos](https://www.uu.nl/staff/MGdeVos)|
| The Data Privacy Handbook is a guide to handling personal data in scientific research, in line with European data protection and privacy regulations. The handbook provides tips, guidelines but also concrete tools and approaches for researchers to handle sensitive data in their projects.| ```

**Comment by parisa-zahedi** [comment]
| [Agri-activism]( https://github.com/UtrechtUniversity/agri_activism) | 
| --- | 
| 2019-2020 |
| Faculty of Social Sciences|
| Technologies: Python, NLP, topic-modeling, network-analysis | 
| RSEs: [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), [Martine de Vos]( https://www.uu.nl/medewerkers/MGdeVos)  |
| This project aims to explore the potential of Twitter data in making sense of online debates. More specifically, the focus is on the online manifestation of the anti-Monsanto movement and assess tweets to investigate (1) activists’ behaviors and opinions, (2) the shape of their networks, (3) their organization and leadership, and (4) information diffusion patterns. Monsanto is one of the world’s largest producers of both agrochemicals and genetically modified crop seeds |

**Comment by parisa-zahedi** [comment]
| [hist-aware]( https://github.com/UtrechtUniversity/hist-aware) | 
| --- | 
| 2020-2021 |
| Faculty of Humanities, History and Art History |
| Technologies: Python, NLP, Deep learning, Huggingface Transformers | 
| RSEs:  [Leonardo Vida]( https://github.com/leonardovida) ,[Parisa Zahedi](https://www.uu.nl/staff/PZahedi)  |
| This project makes use of the Delpher archive (delpher.nl/kranten), which is the largest public collection of digitized pages from Dutch historical newspapers. The research team is mining articles’ sentiments, as expressed by the author of the articles, extracting all the relevant Delpher articles around specific topics (i.e. energy) and is currently training natural language processing (NLP) models called Transformers to extract a sufficiently accurate representation of the sentiment of each article. Currently, the team is making use of the period 1960-1995 consisting of around 250.000 articles around the topics chosen. |

**Comment by parisa-zahedi** [comment]
| [tweet_collector]( https://github.com/UtrechtUniversity/tweet_collector) | 
| --- | 
| 2021 |
| Faculty of Humanities,  Media and Culture Studies |
| Technologies: Python, searchtweets, elasticsearch/kibana | 
| RSEs: [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), [Roos Voorvaart]( https://github.com/RVoor)  |
| Twitter forms a rich source of information for researchers interested in studying 'the public conversation'. The Academic Research product track is designed to serve the needs of the academic research community. It provides researchers with special levels of access to public Twitter data without any cost. This project is aimed to help researchers to use the Academic Research product track to collect tweets of their interest and analyze them. |

**Comment by parisa-zahedi** [comment]
| [Dynamiek in beeld]( https://github.com/sodascience/empathy-viz) | 
| --- | 
| 2021 |
| ODISSEI Social Data Science Team (SoDa) |
| Technologies: Shiny | 
| RSEs: [Parisa Zahedi](https://www.uu.nl/staff/PZahedi), [Shiva Nadi]( https://www.uu.nl/medewerkers/SNadiNajafabadi)  |
| An application that can be used in a clinical setting to score dynamics in empathy. There are some questions to be asked whereafter the results are visualized. The visualization should help the clinician to ask the right questions immediately. |

**Comment by jelletreep** [comment]
I will add the last ones to the list and then will close this issue. New projects can be added using pull requests!

**Comment by XiaSteverding** [comment]
While  I was reviewing articles, suddenly ASReview disappeared.  what should I do to get it back without restart my laptop?

**Comment by qubixes** [comment]
@XiaSteverding I think you're in the wrong place for that question (and for sure in the wrong issue). If you want help with ASReview/you think there is a bug with it, you can try in the official ASReview repository (https://github.com/asreview/asreview/issues). They would probably appreciate it if you are a bit more specific in what happened and what you did.
